addons/test_mass_mailing/tests/test_mailing_test.py

```diff
@@ -59,3 +60,49 @@ def test_mailing_test_button(self):
         })
         with self.mock_mail_gateway(), self.assertRaises(Exception):
             mailing_test.send_mail_test()
+
+    def test_mailing_test_equals_reality(self):
+        """
+        Check that both test and real emails will format the qweb and inline placeholders correctly in body and subject.
+        """
+        contact_list = self.env['mailing.list'].create({
+            'name': 'Testers',
+            'contact_ids': [Command.create({
+                'name': 'Mitchell Admin',
+                'email': 'real@real.com',
+            })],
+        })
+
+        mailing = self.env['mailing.mailing'].create({
+            'name': 'TestButton',
+            'subject': 'Subject {{ object.name }} <t t-out="object.name"/>',
+            'preview': 'Preview {{ object.name }} <t t-out="object.name"/>',
```

Comment:
Hmm, I didn't find a way to easily test the preview, so maybe I should remove this line
```suggestion
```